testing/scenario_app/ios/Scenarios/ScenariosUITests/iPadGestureTests.m

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <XCTest/XCTest.h>
+
+static const NSInteger kSecondsToWaitForFlutterView = 30;
+
+@interface iPadGestureTests : XCTestCase
+
+@end
+
+@implementation iPadGestureTests
+
+- (void)setUp {
+  [super setUp];
+  self.continueAfterFailure = NO;
+}
+
+#ifdef __IPHONE_15_0
```

Comment:
I don't think you want conditional compilation here, you want runtime checks like you have below.

Reply:
I'm not sure this will compile on Xcode 12 without this conditional, since supportsPointerInteraction was only introduced in the Xcode 13 beta.

Reply:
If you want to do that use `#if (defined (__IPHONE_15_0) && __IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_15_0)` then.  We don't just care that the SDK has it, but that the target platform has it.

Reply:
A comment wouldn't hurt, a runtime check with `respondsToSelector` would be a bit more apparent and easier to maintain, but a comment will be good enough.